src/tests/inline/testdata/checkers/typeHint.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+class Boo {}
+
+class FooBad {
+    public parent $a; // want `cannot use 'parent' typehint when current class has no parent`
+
+    /** */
```

Comment:
We probably need some way to test one checker per file.
Like in run-filter-match tests.
Otherwise they're both brittle and require such phpdoc stubs.